src/shared/components/rightbar/RightBar.tsx

```diff
@@ -61,49 +61,53 @@ export default class RightBar extends React.Component<IRightBarProps, IRightBarS
     }
 
     private getWhatsNew() {
-        if (!_.isEmpty(AppConfig.serverConfig.skin_right_nav_whats_new_blurb)) {
-            return (
-                <div className="rightBarSection">
-                    <h3>What's New</h3>
-                    <div dangerouslySetInnerHTML={{__html:AppConfig.serverConfig.skin_right_nav_whats_new_blurb!}}></div>
-                </div>
-            );
-        } else {
-            return (
-                <div className="rightBarSection" style={{paddingBottom:20}}>
-                    <h3 style={{borderBottom:0}}>
-                        What's New
-                        <a href="http://www.twitter.com/cbioportal" className="pull-right">
-                            @cbioportal <i className="fa fa-twitter" aria-hidden="true"></i>
-                        </a>
-                    </h3>
-                    <div style={{marginTop:3}}>
-                        <Timeline
-                            dataSource={{
-                                sourceType: 'profile',
-                                screenName: 'cbioportal'
-                            }}
-                            options={{
-                                username: 'cbioportal',
-                                height: '200',
-                                chrome: 'noheader%20nofooter',
-                            }}
-                            onLoad={() => this.setState({twitterLoading:false})}
-                        />
+        if (AppConfig.serverConfig.skin_right_nav_show_whats_new) {
+            if (!_.isEmpty(AppConfig.serverConfig.skin_right_nav_whats_new_blurb)) {
```

Comment:
why not just get rid of skin_right_nav_whats_new_blurb?  Isn't this redundant?

Reply:
@alisman No, the new property is a boolean which allows to show or not the "What's new" section, while the current property (`skin_right_nav_whats_new_blurb`) defines the content of the section.

Reply:
i see.  ok, we essentially have a default (the twitter thing)